src/svg-renderer.js

```diff
@@ -79,6 +79,12 @@ class SvgRenderer {
             this._svgDom.documentElement.localName !== 'svg') {
             throw new Error('Document does not appear to be SVG.');
         }
```

Comment:
This change looks similar to https://github.com/LLK/scratch-paint/commit/0e74afc631edc0f04d5f7a6391a6d7462e4b7d9c

1. The paint fix searches the string for XML namespaces to keep from having to parse twice. Is that worth doing, or do you feel like parsing twice is safer?
2. Maybe it makes sense to move all of those other error-preventing checks here eventually

Reply:
1. That looks like a better approach to me. I'll make the change.